Resources/Prototypes/_Harmony/Catalog/Fills/Items/toolboxes.yml

```diff
@@ -6,6 +6,8 @@
   components:
   - type: StorageFill
     contents:
+      - id: ToyRubberDuck
+        amount: 2
```

Comment:
This gives free Ducky Galoshes (https://github.com/ss14-harmony/space-station-14/pull/168) to any janitor with a toolbox and galoshes. I want the duckie hunt for duckie galoshes to still be at least a minor challenge and not just round start get.

@Unisol please pitch in.

Reply:
Additionally, it already has a chance to spawn the duck on line 26:

```yaml
      - id: ToyRubberDuck
        prob: 0.15
```